Meter.c

```diff
@@ -190,12 +202,16 @@ static const char* const GraphMeterMode_dotsAscii[] = {
 };
 
 static void GraphMeterMode_draw(Meter* this, int x, int y, int w) {
+   assert(x >= 0);
+   assert(w <= INT_MAX - MAXIMUM(0, x));
```

Comment:
```suggestion
   assert(w <= INT_MAX - x);
```

Suffices. `x>=0` has just been checked above.